config/settings/common.py

```diff
@@ -222,15 +224,16 @@
 CELERY_RESULT_SERIALIZER = 'seed_json'
 CELERY_TASK_RESULT_EXPIRES = 18000  # 5 hours
 
+CELERY_IMPORTS = ('seed.energy.meter_data_processor.tasks')
 CELERYBEAT_SCHEDULE = {
-    'Run daily': {
-        'task': 'seed.energy.tasks.aggregate_monthly_data',
-        'schedule': timedelta(seconds=300), # For Demo purpose, should be days=1 in product environment
+    'Run monthly': {
+        'task': 'seed.energy.meter_data_processor.tasks.aggregate_monthly_data',
```

Comment:
I move the tasks.py to avoid circule import situation, tasks.py needs to
call data analyze function in
energy/meter_data_processor/green_button_data_analyser,
and energy/meter_data_processor/green_button_data_analyser needs to call
the monthly data aggregate function in tasks.py.

On Thu, Mar 3, 2016 at 4:03 PM, Nicholas Long <notifications@github.com>
wrote:

> In config/settings/common.py
> <https://github.com/SEED-platform/seed/pull/817#discussion_r54947941>:
>
> >  CELERYBEAT_SCHEDULE = {
> > -    'Run daily': {
> > -        'task': 'seed.energy.tasks.aggregate_monthly_data',
> > -        'schedule': timedelta(seconds=300), # For Demo purpose, should be days=1 in product environment
> > +    'Run monthly': {
> > +        'task': 'seed.energy.meter_data_processor.tasks.aggregate_monthly_data',
>
> I think we have moved this file a couple times. I think the tasks.py file
> for the energy 'app' is in /seed/energy/tasks.py
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SEED-platform/seed/pull/817/files#r54947941>.
>